core/src/com/po/fuck/model/weapons/Gun.java

```diff
@@ -1,13 +1,17 @@
 package com.po.fuck.model.weapons;
 
 import com.badlogic.gdx.math.Vector2;
+import com.po.fuck.model.sprites.WeaponSpriteInfo;
 import com.po.fuck.model.Entity;
 import com.po.fuck.model.Updatable;
 import com.po.fuck.model.lifetime.Manager;
+import com.po.fuck.model.GeometryMisc;
 
 public abstract class Gun extends HandedWeapon implements Updatable {
     protected float cooldown;
     private float cooldownLeft = 0;
+    protected WeaponSpriteInfo info;
```

Comment:
I don't think that we should store all the info, because we have a size already in a `geometryData`, I think that we need only to store `muzzlePosition` and `handsPosition`